aws-s3-private-bucket/main.tf

```diff
@@ -110,11 +110,13 @@ resource "aws_s3_bucket" "bucket" {
     }
   }
 
-  # TODO
-  #   logging {
-  #   target_bucket = ""
-  #   target_prefix = ""
-  # }
+  dynamic "logging" {
+    for_each = var.logging_bucket == null ? [] : [var.logging_bucket]
+    content {
+      target_bucket = var.logging_bucket.name
```

Comment:
Don't know if it's better to do each.name/each.prefix, or leave it like this.

Reply:
yeah, I could go either way as well. I think the each syntax is a little bit more confusing though for an iterator where we'll always have 0 or 1 loops.